aries_cloudagent/revocation/manager.py

```diff
@@ -241,7 +241,14 @@ async def publish_pending_revocations(
                     await txn.commit()
                 await self.set_cred_revoked_state(issuer_rr_rec.revoc_reg_id, crids)
                 if prev:
-                    await issuer_rr_upd.send_entry(self._profile)
+                    issuer_rr_upd.ready_to_send_check()
+                    await issuer.update_revocation_list(  # use issuer to send entry
+                        issuer_rr_upd.tails_public_uri,
+                        issuer_rr_upd.revoc_reg_def,
+                        prev,
+                        curr,
+                    )
+                    await issuer_rr_upd.set_state_active(self._profile)
```

Comment:
State should be managed by the `issuer.update_revocation_list` so, in combination with my earlier comments, I think this can be removed as well.